compiler/parser/chapel.ypp

```diff
@@ -325,6 +325,7 @@
 %nonassoc TPRAGMA
 
 %type <flag> opt_config
+%type <b> access_control
```

Comment:
I don't consider it a big deal but I note that there appears to be a convention that "optional" productions, like access_control, are prefixed with opt_.  I am not overly wedded to this convention but wonder if you overlooked
it or consciously ignored it (which again, would be OK with me).